pyasdf/generic_io.py

```diff
@@ -531,19 +531,23 @@ def read_until(self, delimiter, delimiter_name=None, include=True):
         raise ValueError("{0} not found".format(delimiter_name))
 
     def seek_until(self, delimiter, include=True):
+        last_block = b''
         while True:
             block = self.read_block()
             if not len(block):
                 break
-            index = re.search(delimiter, block)
+            blocks = last_block + block
+            index = re.search(delimiter, blocks)
```

Comment:
Okay, I see now from the rest of the code where this comes from.  I think maybe the concatenate isn't a big deal, but it might make a noticeable difference if seeking a long ways for the delimiter in question.  I'm curious now to see what difference it makes :)